.eslintrc

```diff
@@ -103,7 +103,11 @@
                 "jsdoc/require-returns-description": "off",
                 "jsdoc/require-param-description": "off",
                 "max-nested-callbacks": ["warn", 5],
-                "sonarjs/cognitive-complexity": "off"
+                "sonarjs/cognitive-complexity": "off",
+                "jsdoc/require-param": "off",
+                "jsdoc/require-returns": "off",
+                "jsdoc/match-description": "off"
+
```

Comment:
Why are you disabling those?

Reply:
Ah, OK. Got it, thanks.

Reply:
Aehm, no. Why do you need to disable those for your tests?

Reply:
@tobiasqueck since this is only test file, we will work on those in a different PR after alignment on what rules should be switched off and which rules should not. I hope this is fine with you. Thanks.

Reply:
I am sorry, but no. If you need the exception for a specific project temporarily, then put them there, but do not put them as global rules in the root folder.